persistence/src/commonMain/db_user/com/wire/kalium/persistence/Users.sq

```diff
@@ -280,3 +280,7 @@ UPDATE User SET team = ? WHERE qualified_id = ?;
 selectNameByMessageId:
 SELECT name FROM User
 WHERE qualified_id = (SELECT Message.sender_user_id FROM Message WHERE Message.id = :messageId AND Message.conversation_id = :conversationId);
+
+countUsers:
+SELECT COUNT(*)
```

Comment:
connection_status have no index and it can make this query very slow

Reply:
Added index on the `connection_status` column